supplementary_style_guide/style_guidelines/mananged-services.adoc

```diff
@@ -0,0 +1,38 @@
+[[managed-services]]
```

Comment:
@cbudz There is a typo in the name of this file (mananged), so when you uncomment in the master.adoc file, it doesn't build. Can you fix the typo?